charts/openfga/values.schema.json

```diff
@@ -2,6 +2,9 @@
     "$schema": "http://json-schema.org/schema#",
     "type": "object",
     "properties": {
+        "global": {
```

Comment:
@jon-whit not sure if relaxing the the top level `additionalProperties` is better than this.

Reply:
We added the strict `additionalProperties: false` to the top level so that it forces us to document the fields which gives us better assurances that our doc on ArtifactHub evolves accurately with changes to the chart as well.

If we drop that strict validation it would alleviate this, but then we'd be back to having to keep us always in check on having matching values and definitions in the values.schema.json file.

One thing we could do is just introduce a CI/CD step that checks the `values.yaml` file against the `values.schema.json` file (with the strict validation), but drop the strict validation in the official json schema. WDYT?

Reply:
I think for now we add `global` because it's a common top-level value in Helm, and if we want to adjust the strictness of the `additionalProperties` in the near future we can revisit that.